DTXMania/Code/Stage/04.Config/CActConfigList.cs

```diff
@@ -904,68 +889,69 @@ public void tSetupItemList_Drums()
                 "表示可能な最小コンボ数（ドラム）：\n" +
                 "画面に表示されるコンボの最小の数\n" +
                 "を指定します。\n" +
                 "1 ～ 99999 の値が指定可能です。",
-                "Initial number to show the combo\n" +
-                " for the drums.\n" +
-                "You can specify from 1 to 99999.");
+                "Set the minimum number of continuous chips \n" +
+                "required to start a drum combo.\n" +
+                "You can select from 1 to 99999.");
             this.listItems.Add(this.iSystemMinComboDrums);
             
             this.iDrumsHHOGraphics = new CItemList("HHOGraphics", CItemBase.EPanelType.Normal, (int)CDTXMania.ConfigIni.eHHOGraphics.Drums,
                 "オープンハイハットの表示画像を変更します。\n" +
                 "A: DTXMania元仕様\n" +
                 "B: ○なし\n" +
                 "C: クローズハットと同じ",
-                "To change the graphics of open hihat.\n" +
-                "A: default graphics of DTXMania\n" +
-                "B: A without a circle\n" +
-                "C: same as closed hihat",
+                "Change the open hihat symbol.\n" +
+                "Type A: Default DTXMania symbol.\n" +
+                "Type B: No circle.\n" +
+                "Type C: Same as the closed HiHat",
                 new string[] { "Type A", "Type B", "Type C" });
             this.listItems.Add(this.iDrumsHHOGraphics);
 
             this.iDrumsLBDGraphics = new CItemList("LBDGraphics", CItemBase.EPanelType.Normal, (int)CDTXMania.ConfigIni.eLBDGraphics.Drums,
                 "LBDチップの表示画像を変更します。\n" +
                 "A: LPと同じ画像を使う\n" +
                 "B: LBDとLPで色分けをする",
-                "To change the graphics of left bass.\n" +
-                "A: same as LP chips\n" +
-                "B: In LP and LBD Color-coded.",
+                "Change the left bass symbol.\n" +
+                "Type A: Same as LP chips\n" +
+                "Type B: In LP and LBD Color-coded.",
                 new string[] { "Type A", "Type B" });
             this.listItems.Add(this.iDrumsLBDGraphics);
 
             this.iDrumsJudgeLinePos = new CItemInteger("JudgeLinePos", 0, 100, CDTXMania.ConfigIni.nJudgeLine.Drums,
                 "判定ラインの位置を\n" +
                 "調整できます。\n" +
                 "0～100の間で指定できます。",
-                "To change the judgeLinePosition for the\n" +
-                "You can set it from 0 to 100.");
+                "Offset the position of the judgment line.\n" +
+                "You can select from 0 to 100.");
             this.listItems.Add(this.iDrumsJudgeLinePos);
 
             this.iDrumsShutterInPos = new CItemInteger("ShutterInPos", 0, 100, CDTXMania.ConfigIni.nShutterInSide.Drums,
                 "ノーツ出現側の\n"+
                 "シャッター位置を調整し\n" +
                 "ノーツの見える位置を\n"+
                 "制限します。",
-                "\n" +
-                "\n" +
-                "");
+                "For Hidden and Sudden play, adjust the \n" +
```

Comment:
I felt that I should clarify what this setting related to because it confused me when setting up initially.

Reply:
This option adjusts the shutter window (which obscures the stage area where notes are falling/raising) for when either hidden or sudden modes are activated.